cac-prod.yml

```diff
@@ -52,4 +52,16 @@ jenkins:
           ^HMCTS_.*IAC.*\/(ia-case-api|ia-shared-infrastructure|ia-ccd-e2e-tests)\/master
         name: IA
         recurse: true
-        title: I & A Dashboard
\ No newline at end of file
+        title: I & A Dashboard
+    - buildMonitor:
+        includeRegex: >-
+          ^HMCTS_.*probate.*\/(probate-frontend|probate-persistence-service|probate-business-service|probate-submit-service|probate-back-office)\/(master|demo|cnp)
```

Comment:
probate is a capital P 
https://build.platform.hmcts.net/job/HMCTS_Probate/
you don't need to be that specific in the regex,
platform is that specific because it excludes jobs not in platform folders, its unlikely you will have other folders include probate in it
.*probate.* might be just as good for you ;)

Reply:
ok